src/vendor/wordpress/demo/image.twig

```diff
@@ -21,9 +22,7 @@
       />
       <figcaption>Mont Blanc appears—still, snowy, and serene.</figcaption>
     </figure>
-  {%
-    if class == 'alignleft' or class == 'aligncenter' or class == 'alignright'
-  %}
+  {% if alignment in ['alignleft', 'aligncenter', 'alignright'] %}
```

Comment:
Does it make sense to DRY the `['alignleft', 'aligncenter', 'alignright']` array?

Reply:
I think it's fine — it's also worth remembering that this is just the storybook example, so it's not going to affect the real site if someone edited this and broke it.